cep18/src/entry_points.rs

```diff
@@ -71,8 +70,36 @@ pub fn approve() -> EntryPoint {
     EntryPoint::new(
         String::from(APPROVE_ENTRY_POINT_NAME),
         vec![
-            Parameter::new(SPENDER_RUNTIME_ARG_NAME, Address::cl_type()),
-            Parameter::new(AMOUNT_RUNTIME_ARG_NAME, U256::cl_type()),
+            Parameter::new(SPENDER, Key::cl_type()),
+            Parameter::new(AMOUNT, U256::cl_type()),
+        ],
+        CLType::Unit,
+        EntryPointAccess::Public,
+        EntryPointType::Contract,
+    )
+}
+
+pub fn increase_allowance() -> EntryPoint {
+    EntryPoint::new(
+        String::from(INCREASE_ALLOWANCE_ENTRY_POINT_NAME),
+        vec![
+            Parameter::new(OWNER, Key::cl_type()),
+            Parameter::new(SPENDER, Key::cl_type()),
+            Parameter::new(AMOUNT, U256::cl_type()),
```

Comment:
```suggestion
            Parameter::new(SPENDER, Key::cl_type()),
            Parameter::new(AMOUNT, U256::cl_type()),
```

owner should be removed :)